src/themes/admin_default/html/layout_default.html.twig

```diff
@@ -62,17 +89,17 @@
                               <svg class="icon icon-tabler icon-tabler-arrow-back-up">
                                  <use xlink:href="#brand-github" />
                               </svg>
-                              {{ 'Source code'|trans }}
+                              <span class="only-large-screens">{{ 'Source code'|trans }}</span>
```

Comment:
I think it is best to show the text from 1024px onwards as there is quite enough space for both icon texts.